itests/eth_filter_test.go

```diff
@@ -291,7 +290,7 @@ func TestEthNewFilterDefaultSpec(t *testing.T) {
 				paddedEthBytes([]byte{0x33, 0x33}),
 				paddedEthBytes([]byte{0x44, 0x44}),
 			},
-			Data: paddedEthBytes([]byte{0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88}),
+			Data: []byte{0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77, 0x88},
```

Comment:
This expectation was wrong, as far as I can tell. The _topics_ are always 32 bytes but the _data_ is arbitrary.